diff --git a/node_modules/jsmind/js/jsmind.js b/node_modules/jsmind/js/jsmind.js
index be0f0b2..81386ef 100644
--- a/node_modules/jsmind/js/jsmind.js
+++ b/node_modules/jsmind/js/jsmind.js
@@ -6,7 +6,7 @@
  *   https://github.com/hizzgdev/jsmind/
  */
 
-;(function($w){
+export default (function($w){
     'use strict';       
     // set 'jsMind' as the library name.
     // __name__ should be a const value, Never try to change it easily.
@@ -2919,5 +2919,8 @@
     } else {
         $w[__name__] = jm;
     }
+
+    return jm;
+
 })(typeof window !== 'undefined' ? window : global);
 
